group_vars/all.yml

```diff
@@ -105,6 +105,9 @@ comics_root: "{{ samba_shares_root }}/comics"
 # Where photos are stored
 photos_root: "{{ samba_shares_root }}/photos"
 
+# Where paperless_ng consumes documents
+consume_documents: "{{ samba_shares_root }}/consume_documents"
```

Comment:
This should really be `documents_root` and "consume" could be a subdirectory of that. The various roots defined are types of media, ie music, movies, comics etc.